build.sh

```diff
@@ -668,7 +668,9 @@ case "$COMMAND" in
         ;;
 
     verify-cocoapods-ios-dynamic)
-        REALM_TEST_BRANCH="$sha" ./build.rb ios cocoapods
+        REALM_TEST_BRANCH="$sha"
```

Comment:
This is a no-op that sets the env variable for the command following it on the line (which is nothing).

Reply:
Changed this, added navigation to the `example/installation` directory given that the CI was failing because it was not able to find the `build.rb`. script